src/main/kotlin/org/rust/lang/RustSdkType.kt

```diff
@@ -24,9 +24,15 @@ class RustSdkType : SdkType("Rust SDK") {
     override fun suggestHomePath(): String? {
         if (SystemInfo.isUnix) {
             // Check whether there is multi-rust installation
-            val multi = File(FileUtil.expandUserHome("~/.multirust/toolchains"))
-            if (multi.exists())
-                return multi.absolutePath
+            val root = "~/.multirust/toolchains"
+            if (File(FileUtil.expandUserHome(root)).exists()) {
+                listOf("stable", "beta", "nightly").forEach {
+                    val multiSub = File(FileUtil.expandUserHome(root.concat(it)))
```

Comment:
Ah yes, that's much better. I haven't worked with File very much before. I pushed a new commit with those changes.